### DISCUSSION TITLE
REQ: <r_shadows model exclusion list>

### DISCUSSION THREAD

**Discussion by timbergeron** [ezQuake/ezquake-source#493] [Ideas]
It's a nice effect to have shadows under item pickups but I bit distracting when shooting projectiles. The option would add an exclusion list for models. Perhaps also an opacity option. 

See image below of shadow of rocket in air:

![ezquake001](https://user-images.githubusercontent.com/68674590/106366999-5c005500-62f4-11eb-8995-4fcf1a751a59.jpg)

**Comment by Calinou** [comment]
I like having this shadow personally as it makes it easier to see how far the projectile is from the ground/a wall. Without shadows, you have no visual indication for that.

(I'd go as far as allowing shadows for players like Quake 2 source ports do, but I know this isn't going to happen :stuck_out_tongue:)

**Comment by sofiauzrlhKeister** [comment]
The shadow effect under item pickups looks great and adds depth to the scene. However, it becomes distracting when applied to fast projectiles like rockets. It would be helpful to include an exclusion list for certain models so effects can be limited to appropriate objects. An additional opacity setting could also allow users to fine-tune the visibility of shadows to better suit gameplay.

**Comment by ChloeNguyen8156** [comment]
Ich verstehe deinen Punkt gut, der Effekt sieht zwar grundsätzlich schön aus, kann aber beim Gameplay tatsächlich ablenken, vor allem bei schnellen Projektilen. Gerade in einem schnellen Shooter wie Quake ist visuelle Klarheit wichtiger als zusätzliche Details. Eine Exclusion-Liste für bestimmte Modelle wäre daher eine sinnvolle Lösung, damit wichtige Objekte [betonredcasino-de.de](https://betonredcasino-de.de/) wie Raketen sauber sichtbar bleiben. Eine zusätzliche Option zur Anpassung der Schatten-Deckkraft wäre ebenfalls hilfreich, um den Effekt individuell feinjustieren zu können.

**Comment by EverettPrice123** [comment]
Concordo, sombras sob itens podem melhorar a percepção de profundidade, mas podem distrair durante combates com projéteis. Ter uma lista de exclusão para modelos específicos permitiria manter sombras apenas [estorilsolcasino](https://estorilsolcasino.com/) em objetos que realmente beneficiam a visibilidade. Além disso, uma opção de ajustar a opacidade das sombras seria útil, permitindo equilibrar efeito visual e clareza durante a jogabilidade, sem comprometer a experiência em momentos de ação intensa.

**Comment by JesseReed123** [comment]
Entiendo tu punto. Tener sombras bajo los objetos recogibles es un efecto visual agradable, pero como señalas, puede distraer al disparar proyectiles, especialmente cuando las sombras siguen a cohetes en el aire como se ve en la [roobet argentina](https://roobett.org/) imagen. Una buena solución sería agregar una lista de exclusión para ciertos modelos, de manera que solo algunos objetos generen sombra. Además, permitir ajustar la opacidad de las sombras podría ayudar a mantener el efecto visual sin interferir con la jugabilidad ni la visibilidad de los proyectiles.
